integration/v2_http_kv_test.go

```diff
@@ -182,7 +182,11 @@ func TestV2CreateUpdate(t *testing.T) {
 	}
 
 	for i, tt := range tests {
-		resp, _ := tc.PutForm(fmt.Sprintf("%s%s", u, tt.relativeURL), tt.value)
+		resp, err := tc.PutForm(fmt.Sprintf("%s%s", u, tt.relativeURL), tt.value)
+		if err != nil {
+			t.Errorf("#%d: put err = %v, want nil", i, err)
```

Comment:
generally if there is etcd failure, it cannot get response in the future too. changed.